fixtures/test-subworkflow/depth-1_workflow-1.yaml

```diff
@@ -12,7 +12,7 @@ input:
 steps:
   wait_1:
     plugin:
-      src: quay.io/arcalot/arcaflow-plugin-wait:atp-v3_cc3e02b
+      src: quay.io/arcalot/arcaflow-plugin-wait:0.2.0
```

Comment:
The fact that someone has to remember to update these four files each time a new release of the plug-in is created feels...unsatisfying...in a couple of different ways.